python/pyspark/sql/catalog.py

```diff
@@ -164,6 +169,47 @@ def listTables(self, dbName: Optional[str] = None) -> List[Table]:
             )
         return tables
 
+    def getTable(self, tableName: str) -> Table:
+        """Get the table or view with the specified name. This table can be a temporary view or a
+        table/view. This throws an AnalysisException when no Table can be found.
```

Comment:
I would follow sphinx synctax in the docs. e.g., class: AnalysisException:

Reply:
let me send a PR to update them.

Reply:
it's a nit. maybe we can fix it togehter when you touch this code :-).

Reply:
sure!